src/HotChocolate/Data/src/NHibernate/publish.cmd

```diff
@@ -0,0 +1,4 @@
+cls
```

Comment:
We need to remember to remove this file before it gets merged. Our script files are located in the root.

Reply:
Removed this filed